src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -66,6 +67,7 @@ public class Swerve extends SubsystemBase {
   SwerveDrivePoseEstimator multitagPoseEstimator;
   SwerveDriveOdometry wheelOdometry; 
   Vision vision;
+  public final NoteCamera noteCamera = new NoteCamera("Microsoft_LifeCam_HD-3000");
```

Comment:
place into its own constant - if used elsewhere, make global and place into a commons folder